apibinding/apibinding/api.py

```
@@ -53,16 +53,101 @@ def _get_response(self, ret_uuid):
+                cause_str = self._error_code_to_string(error.cause, visited, max_depth, _depth + 1)
+                # Only append if we got meaningful content
+                if cause_str:
+                    # Check if it's a special message (circular reference or max depth)
+                    is_special_msg = (cause_str == self._CIRCULAR_REF_MSG or
+                                     cause_str.startswith(self._MAX_DEPTH_MSG_PREFIX))
+                    if not is_special_msg:
+                        msg_parts.append("\n{0}caused by: {1}".format(indent, cause_str))
+                    else:
+                        # For special messages, show them without "caused by:" prefix
+                        msg_parts.append("\n{0}{1}".format(indent, cause_str))
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)